README.md

```diff
@@ -1 +1,182 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Rails Budget App
+
+In this project, I'm building a blog app with rails that will help users create a category and add transactions to it. The user can also see the total amount of money spent and the total amount of money spent per category.
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Rails Blog App](#rails-blog-app)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Blog app ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog app <a name="about-project"></a>
+
+**Budget App** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby and rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Create a new category
+- Create a new transaction
+- See a list of categories
+- See a list of transactions
+- See the total amount of money spent
+- See the total amount of money spent per category
+- Assign a transaction to a category
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 🚀 Live Demo
+
+[Click here to see the page](https://budget-app-xoqr.onrender.com/)
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix the live link sign-up/sign in URL. I got the error in the image below.

![live-app](https://github.com/muhammedb8826/budget-app/assets/53124548/a0a51557-4945-44fb-b2d3-d222d54865df)